test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -1,7 +1,7 @@
 import { Selector, ClientFunction } from 'testcafe';
 
 const TERMINAL_CONTENT = '.o-form-error-container .ion-messages-container';
-const FORM_INFOBOX_ERROR = '.okta-form-infobox-error';
+const FORM_INFOBOX_ERROR = '.o-form-error-container .infobox-error';
```

Comment:
nit: Consider updating these to use data-se attributes https://github.com/okta/okta-signin-widget/blob/14abeb1db905f124765a684cc2f0cc9526331a91/test/testcafe/framework/page-objects/ChallengeOktaVerifyPushPageObject.js#L4